### PR TITLE
News modal fix

### PR DESCRIPTION
**Pull request by MariaPadalka**
Corrected the news creation and editing modality. Added messages (on partners as well). Corrected errors.

### CODE REVIEW

File: src/features/AdminPage/NewsPage/NewsModal/NewsModal.component.tsx [hunk 1]
```diff
@@ -18,215 +30,336 @@ import base64ToUrl from '@/app/common/utils/base64ToUrl.utility';
...
+                                <DatePicker />
+                            </Form.Item>
+                            <PreviewFileModal
+                                opened={previewOpen}
+                                setOpened={setPreviewOpen}
+                                file={filePreview}
+                            />
+                            {/* {invalidFieldsPresent && (
+                                <p className="form-text">!! Заповніть всі обов&apos;язкові поля</p>
+                            )} */}
```
Comment (NadiaKishchuk) [inline src/features/AdminPage/NewsPage/NewsModal/NewsModal.component.tsx:348]: what is the comment?

File: src/features/AdminPage/NewsPage/NewsModal/NewsModal.styles.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+.required-text{
+    cursor: default;
+    margin-bottom: 10px;
+  }
+.form-text{
+    color: c.$ant-menu-text-color;
+    margin-bottom: 5px;
+}
+.image-form-item{
+    margin-top: 25px;
```
Comment (NadiaKishchuk) [inline src/features/AdminPage/NewsPage/NewsModal/NewsModal.styles.scss:20]: please use pxToRem function

**Review by MaksBrat — APPROVED** [review]

File: src/features/AdminPage/NewsPage/NewsModal/NewsModal.component.tsx (hunk 1 above)
Comment (MariaPadalka) [reply src/features/AdminPage/NewsPage/NewsModal/NewsModal.component.tsx]: deleted

File: src/features/AdminPage/NewsPage/NewsModal/NewsModal.styles.scss (hunk 2 above)
Comment (MariaPadalka) [reply src/features/AdminPage/NewsPage/NewsModal/NewsModal.styles.scss:20]: ok. Fixing it

**Review by wehfis — APPROVED** [review]
Excellent work!

**Review by NadiaKishchuk — APPROVED** [review]

**Pull request merged**